go.mod

```diff
@@ -66,7 +66,7 @@ require (
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 )
 
-replace github.com/ethereum/go-ethereum v1.10.16 => github.com/ethereum-optimism/reference-optimistic-geth v0.0.0-20220411203319-ad60590374c8
+replace github.com/ethereum/go-ethereum v1.10.16 => github.com/ethereum-optimism/reference-optimistic-geth v0.0.0-20220426163645-51a763f5c46d
```

Comment:
We should merge the fix commit into the `optimism-prototype` commit, and then use that. It's already in the audit branch anyway, the fix should be in the default branch too.